client/components/videos-ui/index.jsx

```diff
@@ -51,11 +51,23 @@ const VideosUi = ( { headerBar, onBackClick = () => {} } ) => {
 	} );
 
 	useEffect( () => {
+		const videoKeys = Object.keys( course.videos );
```

Comment:
I think you need a null guard here, I wasn't able to open the modal:

<img width="546" alt="Screen Shot 2021-11-25 at 9 04 19 AM" src="https://user-images.githubusercontent.com/349751/143480541-59a3e522-4ccd-4ecb-a48d-e2c561468024.png">

```suggestion
		const videoKeys = Object.keys( course?.videos );
```